package.json

```
@@ -2,36 +2,32 @@
   "name": "@github/textarea-autosize",
   "description": "Autosizes textarea to size of it's contents.",
   "version": "0.1.3",
-  "main": "dist/textarea-autosize.umd.js",
-  "module": "dist/textarea-autosize.esm.js",
-  "types": "index.d.ts",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
   "license": "MIT",
   "repository": "github/textarea-autosize",
   "files": [
-    "dist",
-    "index.d.ts"
+    "dist"
   ],
   "scripts": {
     "clean": "rm -rf dist",
     "lint": "github-lint",
     "prebuild": "npm run clean && npm run lint && mkdir dist",
-    "build": "rollup -c && npm run build-flow",
-    "build-flow": "cp textarea-autosize.js.flow dist/textarea-autosize.esm.js.flow && cp textarea-autosize.js.flow dist/textarea-autosize.umd.js.flow",
+    "build": "tsc",
```

Not having a `UMD` build would be a breaking change. It is not too difficult to output UMD with TS, which would simplify this change. We could change this to output both:

```suggestion
    "build": "tsc --outDir dist/umd/ --module umd && tsc",
```